lab-exercise-1/SarutaFeature.java

```diff
@@ -0,0 +1,5 @@
+public class SarutaFeature {
+    public void feature() {
+        // this is a new feature
```

Comment:
code is empty you should fix

Reply:
I think so.

Reply:
me too

Reply:
it not complete the feature, can you add the feature in your code.

Reply:
don't forget to add your feature